src/git-town-hack

```diff
@@ -15,17 +15,7 @@ function preconditions {
   target_branch_name=$1
   ensure_has_target_branch
 
-  parent_branch_name=$2
-  if [ -z "$parent_branch_name" ]; then
-    parent_branch_name=$MAIN_BRANCH_NAME
-  elif [ "$parent_branch_name" == '.' ]; then
-    parent_branch_name=$INITIAL_BRANCH_NAME
-  fi
-  ensure_has_branch "$parent_branch_name"
-  if [ "$parent_branch_name" != "$MAIN_BRANCH_NAME" ]; then
-    ensure_knows_parent_branches "$parent_branch_name"
-  fi
-
+  parent_branch_name=$MAIN_BRANCH_NAME
```

Comment:
I think this variable can be removed and we can just use `$MAIN_BRANCH_NAME` in the steps.